### PR TITLE
IP-2774: Create VN Infographic to Insights Page

### PR DESCRIPTION
**Pull request by danielbk08**
https://ipricegroup.atlassian.net/browse/IP-2774

### CODE REVIEW

**Comment by danielbk08** [comment]
yes it is. But need to change the image in mobile. the text is small. I'll push it later

File: vn/infographics/index.html [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+            $('#languages').change ( function () {
+                $('.lang').toggle();
+            });
+
+            var lang = $('#languages').val();
+            $('.'+lang).show();
+        });
+    </script>
+</body>
+</html>
```
Comment (rd4704) [inline vn/infographics/index.html]: missing line ending

File: vn/infographics/index.html [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+        <!--<script> dataLayer = [{-->
+            <!--'gaContentGroup': 'static-insights'-->
+        <!--}];-->
+        <!--(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':-->
+            <!--new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],-->
+        <!--j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=-->
+        <!--'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);-->
+        <!--})(window,document,'script','GTM-NMZ8N9');-->
+        <!--</script>-->
+    <!-- End Google Tag Manager -->
```
Comment (rd4704) [inline vn/infographics/index.html]: You can get rid of the comment block?

**Comment by macnguyen** [comment]
Check header width, it should scrollable on mobile.
And, should reduce the transition for header to 0.2s for smooth

**Pull request merged**